024-DoYouEvenSynapse/Coach/README.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+
+The coach section is dedicated to those who are running the What The Hack
+
+# Introduction
+
+# Coach Guide
+- Challenge 0: **[Pre-requisites - Ready, Set, GO!](00-prereqs.md)**
+   - Prepare your workstation to work with Azure
+- Challenge 1:  **[Gathering Open Powerlifting Data](01-data-gathering.md)**
+   - Retrieve data and upload to Azure using a storage service
+- Challenge 2:  **[Load data into a relational database/warehouse](02-load-data.md)**
+   - Loading the data
+- Challenge 3:  **[Visualization](03-visualization.md)**
+   - Visualizing the data
+- Challenge 4:  **[Incremental Loads](04-incrementals.md)**
+   - Load incrementals
+- Challenge 5:  **[Mask Data for Privacy](05-data-masking.md)**
+   - Putting your mask on
+ - Challenge 6: **[Add New Data Source(s)](06-new-data.md)**
+   - Additional data sources
+- Challenge 7:  **[Leverage Machine Learning](07-ml.md)**
+   - The machines know too much!
+
+# Coaches Resources
+- Lab Data Prep.ipynb : A Python notebook that splits the Open Powerlifting data into an initial historical data file and a chosen number of daily incremental files.  To be run in a Synapse Workspace.
```

Comment:
Please bold 'Lab Data Prep.ipynb' and remove the space before the semicolon

Reply:
Amended